conferences/2026-01-31_FOSDEM-kernel-devroom.md

```diff
@@ -0,0 +1,66 @@
+# Kernel Devroom @ FOSDEM 26
+
+We are pleased to announce the Call for Participation (CfP) for the FOSDEM 2026 Kernel Devroom.
+
+FOSDEM 2026 will be over the weekend of the 31st of January and 1st of February in Brussels, Belgium.
+FOSDEM is a free and non-commercial event organised by the community for the community.
+The goal is to provide free and open source software developers and communities a place to meet to:
+
+- get in touch with other developers and projects;
+- be informed about the latest developments in the free software world;
+- be informed about the latest developments in the open source world;
+- attend interesting talks and presentations on various topics by project leaders and committers;
+- to promote the development and benefits of free software and open source solutions.
+- Participation and attendance is totally free, though the organisers gratefully accept donations and sponsorship.
+
+## Format
+
+The Kernel Devroom will be running an all day, exact day and time will be clarified in the near future..
+
+We’re looking for talk or demo proposals in one of the following 4 sizes:
+
+- 10 minutes (e.g., a short demo)
+- 20 minutes (e.g., a project update)
+- 30 minutes (e.g., introduction to a new technology or a deep dive on a complex feature)
+- 40 minutes (e.g., a deep dive on a complex feature)
+
+In all cases, please allow for at least 5 minutes of questions (10min preferred for the 30min slots).
+In general, shorter content will be more likely to get approved as we want to cover a wide range of topics.
+
+## Proposals
+
+Proposals should be sent through the FOSDEM scheduling system at: https://pretalx.fosdem.org/fosdem-2026/cfp
+
+Please select the "Kernel" as the __track__ and ensure you include the following information when submitting a proposal:
```

Comment:
"the "Kernel"" should likely just be "Kernel"

Reply:
fixed both